database/sql/V056__remove_surplus_fee.sql

```diff
@@ -0,0 +1,8 @@
+-- These field are not needed anymore since they were used to calculate the 
+-- surplus fee for limit orders beforehand. Now, since the surplus fee 
+-- is provided by the solvers at the time of solving, we can remove these feilds 
+-- and rely on the order_execution table to get the executed surplus fee.
+
+ALTER TABLE orders 
+  DROP COLUMN surplus_fee,
+  DROP COLUMN surplus_fee_timestamp;
```

Comment:
ubernit: Missing newline at EOF.

Reply:
Added. Btw, why do we need it?

Reply:
It's not needed per se, but it makes future diffs nicer. Adding code to the end of the file becomes a one line change instead of a two line change (one extra line change to add the `\n` to the end of the last line).